styleguide/templatetags/styleguide_tags.py

```diff
@@ -102,3 +102,8 @@ def get_styleguide_templates():
         templates.append((name, slug, sub_slugs))
 
     return templates
+
+@register.assignment_tag
+def open_menu(path, sub_slugs):
+    return path.split('/')[2] in sub_slugs
```

Comment:
This line fails with an exception if you're on the top level page, ie. the path is `/`

Reply:
It also doesn't seem to return true when you are on the category-level, for example on "Forms" and not its "Input" subsection

Reply:
Good catch on the error.  

This is only supposed to return True when you are on a subsection.  When you are on category level, no submenu should not be open.